.github/workflows/ci.yml

```
@@ -0,0 +1,45 @@
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+jobs:
+  coding-standards:
+    name: "Coding Standards"
+    runs-on: ubuntu-latest
```

Using the latest version might break builds in the future. What about
```suggestion
    runs-on: ubuntu-18.04
```